index.yaml

```diff
@@ -3990,6 +3990,25 @@ entries:
     - assets/rancher-vsphere-csi/rancher-vsphere-csi-2.1.000.tgz
     version: 2.1.000
   rancher-webhook:
+  - annotations:
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/hidden: "true"
+      catalog.cattle.io/namespace: cattle-system
+      catalog.cattle.io/os: linux
+      catalog.cattle.io/release-name: rancher-webhook
+    apiVersion: v2
+    appVersion: 0.2.1-rc1
+    created: "2021-09-15T16:38:32.254789-07:00"
+    dependencies:
+    - condition: capi.enabled
+      name: capi
+      repository: ""
+    description: ValidatingAdmissionWebhook for Rancher types
+    digest: 2144f7ad506cf197d8013bdbf824a881d03b8fc13b50669bcf91389fec0053c0
+    name: rancher-webhook
+    urls:
+    - assets/rancher-webhook/rancher-webhook-1.0.1+up0.2.1-rc1.tgz
+    version: 1.0.1+up0.2.1-rc1
```

Comment:
the version will become `1.0.1+up0.2.1` when the upstream rancher-webhook 0.2.1 is cut.